src/index.html

```diff
@@ -8,11 +8,24 @@
 </head>
 <body>
     <div id="controls" class="hidden">
-      <div id="file-selector">
+      <div id="options-panel">
+        <div>
+          <h3>Options</h3>
+          <label>Fit:
+            <select name="fitTypes" id="fitType-selected">
+              <option value="Default" selected="yes">Default</option>
+              <option value="HorizontalVertical">Horizontal-Vertical</option>
+            </select>
+          </label>
+        </div>
+      </div>
+      <div id="file-panel">
+        <div id="file-selector">
           <input type="file" id="bpmn-file" name="file"/>
           <label for="bpmn-file" class="btn"><span>open BPMN file</span></label>
+        </div>
+        <p class="info">(either drop a file here)</p>
       </div>
-      <div class="info-centered"><p>(either drop a file here)</p></div>
     </div>
     <div id="graph"></div>
```

Comment:
as already explained, fixed viewport is required to make the fit fully work. For instance with our `model-complete-semantic.bpmn` test diagram, the diagram fits only horizontally and not vertically (vertical scrollbars are displayed)
![image](https://user-images.githubusercontent.com/27200110/97329084-f759d800-1876-11eb-8cb6-d5100388b96d.png)

Reply:
in a page with fixed viewport like 'non-regression.html', the diagram fits correctly

![image](https://user-images.githubusercontent.com/27200110/97330466-903d2300-1878-11eb-92b1-d74addafdf32.png)

Reply:
**Proposal**: when we select a fit on load option, we apply a new css class to the diagram div to set the width and height.
The 2 css properties can be expressed in pixels, percentages, ... but must be set to have what I call a 'fixed size viewport'
And when we select 'none', we remove the css class to restore the original behaviour of the demo.
This also limit the number of things to update to make the fit feature works in the page

@csouchet what do you think about this?

Reply:
I add to fix clearly the height of the div wrapping the div of 'graph' in the demo page, because the graph container overrided the button and the options panel.

If you find a better idea, I'm open (cc @aibcmars)